ui/src/app/components/inputs.tsx

```diff
@@ -240,8 +240,7 @@ export const TextAreaWithLengthValidationMessage = (
   return (
     <React.Fragment>
       <TextArea
-        {...{ id }}
-        aria-label={id}
```

Comment:
@jmthibault79 auto-populated aria-label here:
https://github.com/all-of-us/workbench/commit/7fbfeac86143b6d29dc26892e8af31c5f3ab5b27#diff-e20d25e18ad70d1c526af33dd200e80376f4c624abc466a553a1563a9c0eb3d7R236